openshift/template.yaml

```diff
@@ -157,6 +157,77 @@ objects:
               secretKeyRef:
                 key: session_secret
                 name: kibana-oauth-application
+- apiVersion: batch/v1
+  kind: Job
+  metadata:
+    name: load-historical-data
```

Comment:
Not using `generateName` on purpose: this way this job can only deployed once. This avoids problems in case something goes wrong and the app would get deployed again before reverting this change.